api/src/main/java/pl/kolendateam/dadcard/classCharacter/entity/ClassCharacter.java

```diff
@@ -0,0 +1,33 @@
+package pl.kolendateam.dadcard.classCharacter.entity;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@RequiredArgsConstructor
+@Entity
+public class ClassCharacter {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    int id;
+
+    @NonNull
+    private String type;
```

Comment:
zrobil bym to jako nie jako String tylko enum

Reply:
Zmienię na enum, ok